### DISCUSSION TITLE
Start with Windows or at User Logon

### DISCUSSION THREAD

**Discussion by katastrophe92** [espanso/espanso#1498] [Q&A] [answered]
How to do it? I always forget to start it when i need it. Tried also to make a rule in Task Scheduler but no success

**Comment by ptrkhh** [comment, marked as answer]
Just create a shortcut and put it in C:\ProgramData\Microsoft\Windows\Start Menu\Startup (for all users) or C:\Users\<YOUR USERNAME>\AppData\Microsoft\Windows\Start Menu\Startup
